### PR TITLE
Next release

### PR DESCRIPTION
**Pull request by andrewnicols**
This commit adds the following changes:
- a new version management script to make modifying the versions.json far easier
- a fix for some 1.9 version data
- the next set of release information for each branch

### CODE REVIEW

File: scripts/version.mjs [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+    });
+
+    return major;
+};
+
+const getReleaseDate = async (defaultDate = null) => (
+    await inquirer.prompt({
+        type: 'input',
+        name: 'releaseDate',
+        message: 'What is the expected release date?',
```
Comment (junpataleta) [inline scripts/version.mjs:112]: It might be good to specify acceptable date formats.

File: scripts/version.mjs [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+        let calculatedDate;
+        calculatedDate = new Date(new Date(releaseDate).setMonth(releaseDate.getMonth() + supportLength));
+        calculatedDate = new Date(calculatedDate.setDate(calculatedDate.getDate() - 1));
+        return getFormattedDate(calculatedDate);
+    };
+
+    const { theDate } = await inquirer.prompt({
+        type: 'input',
+        name: 'theDate',
+        message: 'What is the general support end date?',
```
Comment (junpataleta) [inline scripts/version.mjs:213]: Should be "security support end date"

**Review by junpataleta — CHANGES_REQUESTED** [review]
Thanks, @andrewnicols! The version script is really handy! 

It just needs some minor changes from my review comments.

I'd also suggest we document this script either as part of
- [Task 9 (release notes task) of the 1 week prior](https://moodledev.io/general/development/process/release#1-week-prior) release tasks
- [Task 4 of the Release day tasks](https://moodledev.io/general/development/process/release#release-day).

What do you think?

**Comment by andrewnicols** [comment]
I've updated with the requested changes and improved the help for majors.

> I'd also suggest we document this script either as part of

I agree

> * [Task 9 (release notes task) of the 1 week prior](https://moodledev.io/general/development/process/release#1-week-prior) release tasks

Usually Mick is the one who does this. @mickhawkins which step do you do this in? We should try and document it.

> * [Task 4 of the Release day tasks](https://moodledev.io/general/development/process/release#release-day).
> 
> What do you think?
That step is already wrong. Again I think it's done by Mick about a day or two before a release when he does the release notes.

**Review by junpataleta — APPROVED** [review]
Thanks, Andrew! Looks good!

**Pull request merged**